src/components/Text/Text.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+/* @jsx h */
+import {h} from 'hyperapp'
+import cn from 'classnames'
+import styles from './Text.css'
+
+import type {Props} from './flow-typed'
+import type {Children} from 'hyperapp'
+
+export const Text = ({className, isMuted, isPre, size}: Props, children: Array<Children>) => (
+  <div className={cn(
+    styles.text,
+    isMuted && styles.text_muted,
+    isPre && styles.text_pre,
```

Comment:
I think we can don't create pre version for Text. We'll create Code component someday